quality_control_stock/tests/test_quality_control_stock.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Oihane Crucelaegui - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp.tests.common import TransactionCase
+
+
+class TestQualityControl(TransactionCase):
+
+    def setUp(self):
+        super(TestQualityControl, self).setUp()
+        self.picking_model = self.env['stock.picking']
+        self.operation_model = self.env['stock.pack.operation']
+        self.qc_trigger_model = self.env['qc.trigger']
+        self.product = self.env.ref('product.product_product_4')
+        self.partner = self.env.ref('base.res_partner_2')
+        self.test = self.env.ref('quality_control.qc_test_1')
+        self.picking_type = self.env.ref('stock.picking_type_out')
+        self.trigger = self.qc_trigger_model.search(
+            [('picking_type', '=', self.picking_type.id)])
+        move_vals = {
+            'name': self.product.name,
+            'product_id': self.product.id,
+            'product_uom': self.product.uom_id.id,
+            'location_id': self.picking_type.default_location_src_id.id,
+            'location_dest_id': self.picking_type.default_location_dest_id.id,
+        }
+        self.picking1 = self.picking_model.create({
+            'partner_id': self.partner.id,
+            'picking_type_id': self.picking_type.id,
+            'move_lines': [(0, 0, move_vals)],
+        })
+        self.picking1.action_confirm()
+        self.picking1.action_assign()
+        self.picking1.do_prepare_partial()
+        for line in self.picking1.move_lines:
+            self.operation_model.create({
```

Comment:
You have to instantiate `stock.transfer_details` TransientModel with `context = {'active_ids': [self.picking1.id], 'active_id': self.picking1.id}`, and execute `do_detailed_transfer`